test/manage/components/item-test.js

```diff
@@ -2,11 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/* global describe, before, after, beforeEach, afterEach, it */
```

Comment:
There is an eslint plugin for mocha, and eslint is intelligent enough in locating configuration files that you can have a tests-specific .eslintrc.js(on) that extends a core .eslintrc file.

That would simplify the boilerplate on tests.

Reply:
Cool, I'll look into that. I'll probably do that in a followup though, since my stack of PRs is getting dangerously tall...

Reply:
Tests-specific eslint is what `lockbox-datastore` does.  It is really easy...

Reply:
Ok, I'll just copy that. It shouldn't cause too many rebasing issues for my subsequent PRs...

Reply:
Added #60 to track it